Formula/convox.rb

```diff
@@ -0,0 +1,15 @@
+class Convox < Formula
+  desc "The convox AWS PaaS CLI tool"
+  homepage "https://convox.com/"
+  url "https://bin.equinox.io/a/9RUvivxx6Lm/convox-20160606232025-darwin-amd64.tar.gz"
```

Comment:
Sure thing!

The larger code base is here: https://github.com/convox/rack/

However the CLI tool (which this is installing) is located here: https://github.com/convox/rack/tree/master/cmd/convox

Hope that helps!